docs/UserGuide.adoc

```diff
@@ -425,10 +479,18 @@ ot/ORDER_TYPE,tid/TRANSACTION_ID, n/NAME, a/ADDRESS, p/PHONE_NUMBER, e/EMAIL, rt
 * Duplicate and invalid order or return order will not be imported into Delino.
 ====
 
-==== Examples
+[[import-combination]]
+There are only one possible combination for the `import` Command:
+
+. Possible combination of Import command
```

Comment:
Need to remove the space for `. Possible combination of Import command`, or else the table name will not be shown.
Should be `.Possible combination of Import command` instead

![image](https://user-images.githubusercontent.com/38811217/77912344-fc48b000-72c4-11ea-8771-45cc1d4ad290.png)

Reply:
Done! Thanks for the look out!